CombinatorialGames/Game/Concrete.lean

```diff
@@ -1,136 +1,201 @@
 /-
 Copyright (c) 2025 Violeta Hernández Palacios. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Violeta Hernández Palacios
+Authors: Violeta Hernández Palacios, Junyan Xu
 -/
-import CombinatorialGames.Game.IGame
+import CombinatorialGames.Game.Loopy.IGame
 import CombinatorialGames.Game.Impartial.Basic
 
 /-!
 # Combinatorial games from a type of states
 
-A "concrete" game is a type of states endowed with well-founded move relations for the
-left and right players. This is often a more convenient representation for a game, which can then be
-used to define a `IGame`.
+In the literature, mathematicians often describe games as graphs, consisting of a set of states, as
+well as move relations for the left and right players. We define a structure `ConcreteGame` which
+facilitates this construction, bundling the left and right set functions along with the type, as
+well as functions `ConcreteGame.toLGame` and `ConcreteGame.toIGame` which turn them into the
+appropriate type of game.
+
+Mathematically, `ConcreteGame.toLGame` is nothing but the corecursor on loopy games, while
+`ConcreteGame.toIGame` is defined inductively.
+
+## Design notes
+
+When working with any "specific" game (nim, domineering, etc.) you can use  `ConcreteGame` to set up
+the basic theorems and definitions, but the intent is that you're not working with `ConcreteGame`
+directly most of the time.
+
+If you want to prove theorems about a particular game, you should organize your material as follows:
+
+- Any preliminaries about the game (definitions, structures, etc.) go in a namespace inside
+  `ConcreteGame`, e.g. `ConcreteGame.Poset`, `ConcreteGame.Domineering`, etc.
+- Theorems about the actual `IGame`s or `LGame`s should go in the corresponding namespace.
```

Comment:
may need to be changed

Reply:
Yep, that's why I turned this into a draft. Will update once I get back home.